lib/idp/constants.rb

```diff
@@ -94,7 +94,10 @@ module Vendors
     AAL2 = 2
     AAL3 = 3
 
+    MOCK_IDV_APPLICANT_FULL_STATE = 'Montana'
+    MOCK_IDV_APPLICANT_FULL_STATE_ID_JURISDICTION = 'North Dakota'
     MOCK_IDV_APPLICANT_STATE_ID_JURISDICTION = 'ND'
+    MOCK_IDV_APPLICANT_STATE = 'MT'
```

Comment:
I think `MOCK_IDV_APPLICANT_STATE` and `MOCK_IDV_APPLICANT_STATE_ID_JURISDICTION` can both be deleted. We can/should get both from MOCK_IDV_APPLICANT.

I think the other two constants are needed (MOCK_IDV_APPLICANT_FULL_STATE, MOCK_IDV_APPLICANT_FULL_STATE_ID_JURISDICTION) because we don't store the full state on mock idv applicant. 

What do you think?